packages/kbn-es-archiver/src/cli.ts

```diff
@@ -144,7 +144,7 @@ export function runCli() {
 
         const query = flags.query;
         let parsedQuery;
-        if (typeof query === 'string') {
+        if (typeof query === 'string' && query.length > 0) {
```

Comment:
This can perhaps be simplified as `if (!query) {}`, but I also like the explicitness of what you have.

Reply:
Thanks! I can change it to your proposal. It's up to you :)